apigee_edge.install

```diff
@@ -87,6 +87,15 @@ function apigee_edge_install() {
     ];
     user_role_grant_permissions(RoleInterface::AUTHENTICATED_ID, $authenticated_user_permissions);
   }
+
+  // Set default client configuration.
+  \Drupal::state()->setMultiple([
```

Comment:
Why do not you use one `apigee_edge.client` state key (name) and serialize all these configurations under it? Currently you create multiple entries to store all client config.